lxc/exec_unix.go

```diff
@@ -42,6 +42,11 @@ func (c *cmdExec) controlSocketHandler(control *websocket.Conn) {
 		sig := <-ch
 		switch sig {
 		case unix.SIGWINCH:
+			if !c.interactive {
+				// Don't send SIGWINCH to non-interactive, this can lead to console corruption/crashes.
```

Comment:
Thanks for this comment, this should help a reoccurrence of the regression in the future by drawing attention to the detail.